frontend/packages/ceph-storage-plugin/src/utils/install.ts

```diff
@@ -58,12 +58,30 @@ export const getAssociatedNodes = (pvs: K8sResourceKind[]): string[] => {
   return Array.from(nodes);
 };
 
-export const shouldDeployMinimally = (nodes: NodeKind[]) =>
+export const shouldDeployInternalAsMinimal = (nodes: NodeKind[]) => {
+  const { totalCPU, totalMemory } = nodes.reduce(
+    (acc, curr) => {
+      const cpus = humanizeCpuCores(getNodeCPUCapacity(curr)).value;
+      const memoryRaw = getNodeAllocatableMemory(curr);
+      const memory = humanizeBinaryBytes(convertToBaseValue(memoryRaw)).value;
+      acc.totalCPU += cpus;
+      acc.totalMemory += memory;
+      return acc;
+    },
+    {
+      totalCPU: 0,
+      totalMemory: 0,
+    },
+  );
+  return totalCPU < 30 || totalMemory < 60;
```

Comment:
We have different values over here for total memory & the message is saying 72 GiB. We need to be consistent.

Reply:
@cloudbehl We are putting as 24 Gigs per node. I am assuming the worst-case conversion will come to 20 so taking at 60. RAM is not an as hard constraint as CPUs.

Reply:
Understood but the issue is we can have either message say the same or the validation should match because when admin is going to match these numbers will not match.

The message will say 72 Gib required when total of nodes goes below 60. Which I think will be weird & QE will raise bugs.

Reply:
If you have noticed then the memory limit is not exactly like we want it to be sometimes it goes down by 3 to 4 GBs. For ex: 64GiB usually shows up as 61 and as you mentioned in my other PR to even 60 GiB at times. Now I am doing this calculation based ob (24GiB - 4 Error limit) * 3 Nodes. Hope this clears things out for you.

Reply:
The thing that matters is allocatable memory, not the memory provisioned at the time cluster was created. 

Also, memory is something mentioned in the node list table already & admin is going to calculate based on those values.

 I'm just saying user & QE will be confused with these values & I don't see any harm in having the same values and we should do that.

Reply:
Skipping for now as this needs more discussion.